assets/dev/js/editor/controls/media.js

```diff
@@ -243,6 +251,54 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 	onBeforeDestroy() {
 		this.$el.remove();
 	},
+
+	async onMediaImageSize() {
+		this.ui.mediaImageSize.css( 'color', 'inherit' );
```

Comment:
explain

Reply:
Remove the gray color text color when user changes the breakpoint view. 
The default value is with gray color, when the user change the size the gray color is removed.
As the design mentions.

Reply:
I replaced this with class.